chia/rpc/wallet_rpc_api.py

```diff
@@ -541,11 +541,12 @@ async def create_new_wallet(self, request: Dict) -> EndpointResult:
                     my_did_id = encode_puzzle_hash(
                         bytes32.fromhex(did_wallet.get_my_DID()), AddressType.DID.hrp(self.service.config)
                     )
+                    nft_wallet_name = request.get("wallet_name", "") + " NFT Wallet"
```

Comment:
In some way avoid the leading space this introduces.  Here's one of a few options that crossed my mind.

```suggestion
                    request_wallet_name = request.get("wallet_name")
                    if request_wallet_name is None:
                        nft_wallet_name = "NFT Wallet"
                    else:
                        nft_wallet_name = f"{request_wallet_name.strip()} NFT Wallet"
```